common-arcana.lic

```diff
@@ -137,9 +137,33 @@ module DRCA
     activated
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = nil, runestone_tm = false)
+  def prepare?(settings, abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = nil, runestone_tm = false, unleash_item = false)
```

Comment:
Why do you need the settings argument?

Reply:
It's needed for `cast_spell` and doesn't really have any negative effects. I am also planning to just pass data instead of all these parameters, that is more in-keeping with the rest of the library.

Reply:
Looks like prepare is called in many contexts.  I will probably prefer exposing settings globally in common-arcana.lic or whatever the accepted pattern is, rather than passing a new function parameter, although it may be better to revise all existing uses of `prepare?` to be of the form `prepare?(settings, data)` anyways, just like the calling function, so that less special context exists across other scripts.